src/Resources/config/services/data_mapper.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://symfony.com/schema/dic/services"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+
+        <service id="setono_sylius_facebook.data_mapper.composite"
+                 class="Setono\SyliusFacebookPlugin\DataMapper\CompositeDataMapper"/>
+
+        <service id="setono_sylius_facebook.data_mapper.order"
+                 class="Setono\SyliusFacebookPlugin\DataMapper\OrderDataMapper">
+            <argument type="service" id="setono_sylius_facebook.formatter.money" />
+
+            <tag name="setono_sylius_facebook.data_mapper" />
```

Comment:
Maybe give these mapper a priority of `-100` for example. Just so it's easy for plugin users to add a data mapper that overwrites default data set by default mappers

Reply:
Reasonable